fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LucenePartitioner.java

```diff
@@ -739,13 +743,15 @@ private CompletableFuture<Integer> moveDocsFromPartition(@Nonnull final LucenePa
                         .setCount(partitionInfo.getCount() - records.size())
                         .setFrom(ByteString.copyFrom(Tuple.from(newBoundaryTimestamp).pack()));
                 savePartitionMetadata(groupingKey, builder);
-                if (LOGGER.isInfoEnabled()) {
-                    LOGGER.info(KeyValueLogMessage.of("Repartitioning Records",
+                if (LOGGER.isDebugEnabled()) {
+                    LOGGER.debug(KeyValueLogMessage.of("Repartitioning Records",
                             LuceneLogMessageKeys.GROUP, groupingKey,
                             LuceneLogMessageKeys.PARTITION, partitionInfo.getId(),
                             LuceneLogMessageKeys.TOTAL_COUNT, partitionInfo.getCount(),
                             LuceneLogMessageKeys.COUNT, records.size()));
                 }
+                state.context.increment(LuceneEvents.Counts.LUCENE_REBALANCE_PARTITION);
+                state.context.increment(LuceneEvents.Counts.LUCENE_REBALANCE_PARTITION_DOC_COUNT, records.size());
```

Comment:
This can be coalesced into one event with https://github.com/FoundationDB/fdb-record-layer/pull/2519/